### ISSUE TITLE
Running long term forecasting on custom data

### ISSUE THREAD

**Issue by FarahSaeed** [thuml/Time-Series-Library#289]
Hi 

This is regarding running PatchTST for long term forecasting on custom data. The data is related to public influenza weekly cases of a region from Our world in data. Attached file covers the data.  The  [PatchTST.sh](scripts/long_term_forecast/ILI_script/PatchTST.sh) in ILI script is run and the dataset path is replaced, the pred len = 24. All other parameters are same as [PatchTST.sh](scripts/long_term_forecast/ILI_script/PatchTST.sh). The training loss decreases but the validation loss increases in every epoch and due to early stopping, the program stops after few epochs. The learning rate is also updated to 1e-5. but the val loss still increases. With the same parameters the code shows good results for ILI dataset. 

Here are two examples of datasets used. 
[influenza_pakistan.csv](https://github.com/thuml/Time-Series-Library/files/13520107/influenza_pakistan.csv)
[influenza_china.csv](https://github.com/thuml/Time-Series-Library/files/13520108/influenza_china.csv)

Any insights regarding this are highly appreciated.

Thanks!

**Comment by wuhaixu2016** [comment]
Hi, thanks for your experiments.

I think this situation indicates that the model is overfitted. You can add some dropout to attention or MLP layers.
